src/components/dashboard/Mentorship.js

```diff
@@ -48,27 +49,34 @@ class Mentorship extends React.Component {
       all: true,
     }
   }
-  
+
+  componentWillUnmount() {
+    /*** We would like to store filter state in Redux on component unmounting
+     * (when user navigates away), and restore it from state when the component
+     * is mounted again (user navigates back)— thoughts?
+    */
+  }
```

Comment:
@bonham000 I'm thinking we maybe should not do this actually? Or worth talking about I guess, but it's pretty easy to get back to where you were, I think it might be best to start over with a fresh search each time